examples/webpack/webpack.config.js

```diff
@@ -0,0 +1,51 @@
+const path = require('path')
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin')
+
+const config = {
+  mode: 'production',
+  devtool: 'source-map',
+  optimization: {
+    minimizer: [
+      new UglifyJsPlugin({
+        cache: true,
+        parallel: true,
+        sourceMap: true,
+        uglifyOptions: {
+          compress: {
+            ie8: true,
+            warnings: false
+          },
+          mangle: {
+            ie8: true
+          },
+          output: {
+            comments: false,
+            ie8: true
+          }
+        }
+      })
+    ]
+  },
+  entry: './assets/main.js',
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: ['env', 'es3']
+          }
+        }
+      }
+    ]
+  },
+  output: {
```

Comment:
Did you manage to avoid polyfilling the wrapper?

Reply:
yep

Reply:
will add setup for scss and images too, but may be a good idea to wait for until after the restructure happens.